src/store/Test/TestoneActions.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * MÅSTE ALLTID VARA STORA BOKSTÄVER PÅ ACTIONS
```

Comment:
Ta bort, stämmer inte. Det är en konvention för att det är `CONSTANTS`. konstanter ska alltid vara stora bokstäver, men `actions` är ingen magi, det är bara vanliga object, och `type` är bara key-value pair. Du kan ändra `type` till `potatis`, så länge din reducer kollar efter `potatis` istället för `type`. Det ska du givetvis inte göra, men det är viktigt att förstå vad som händer. Den riktiga anledningen till att man använder konstanter är för att ´actionTypes´ är strängar och det är dessa som används för att identifiera actions. Därför gäller det att dem matchar 100%. Det är då lättare att säga att allt ska alltid vara stora bokstäver, så är och constants som ligger centraliserat i en fil (single source of truth) så är chansen att du fuckar upp mindre. Det är väldigt svårt att hitta den typen av bugg nämligen.

Reply:
Tack för informationen, pinnar detta samtidigt i vår Slack chatt.

Reply:
pinnarN^